syntax = "proto3";

message FunctionCall {
  string name = 1;
}